srcpkgs/json-c/template

```diff
@@ -1,21 +1,32 @@
-# Template build file for 'json-c'.
 pkgname=json-c
 version=0.13.1
-revision=1
+revision=2
 build_style=gnu-configure
+configure_args="$(vopt_enable threading) $(vopt_enable rdrand)"
 hostmakedepends="automake libtool"
+checkdepends="valgrind"
 short_desc="A JSON implementation in C"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="MIT"
 homepage="http://oss.metaparadigm.com/$pkgname"
 distfiles="https://s3.amazonaws.com/json-c_releases/releases/$pkgname-$version.tar.gz"
```

Comment:
would be nice if switched to the github distfiles in accordance with the new homepage= 

if possible